techTree.md

```diff
@@ -0,0 +1,97 @@
+# Stateless Tech Tree
+
+A 'roadmap' delineates centrally determined deliverables and milestones. Because the Stateless Ethereum effort is not centrally planned, it's helpful to instead use a [tech tree](https://en.wikipedia.org/wiki/Technology_tree) to organize and record ongoing work in the Stateless Ethereum initiative. This helps to map out a shared understanding of 'the critical path' and dependencies in research / implementation.
+
+![tech_tree](./assets/Stateless_Tech_Tree_06-2020.png)
+
+> ### Key
```

Comment:
What do red nodes mean?

Reply:
They are just the major milestones of Stateless Ethereum... I'll add it to the key

Reply:
Actually, this comment has caused me to re-think using colors at all; after checking this website (https://www.toptal.com/designers/colorfilter) I found that it was really hard to distinguish the colors with the most common types of colorblindness. I'm going to re-work the diagram to be easy on the cones (of the retina).